language/stdlib/transaction_scripts/rotate_authentication_key_with_nonce.move

```diff
@@ -49,5 +51,10 @@ spec fun rotate_authentication_key_with_nonce {
 
     /// This rotates the authentication key of `account` to `new_key`
     include LibraAccount::RotateAuthenticationKeyEnsures{addr: account_addr, new_authentication_key: new_key};
+
+    aborts_with [check]
+        Errors::INVALID_ARGUMENT,
+        Errors::INVALID_STATE,
+        Errors::NOT_PUBLISHED; // TOOD: added due to the possible absence of SlidingNonce in SlidingNonce::try_record_nonce
```

Comment:
I think adding this in makes sense since this occurring before the LR check. Similarly for the admin script below.